### DISCUSSION TITLE
blog/use-custom-latex-macros-to-boost-your-writing-productivity

### DISCUSSION THREAD

**Discussion by giscus** [cesarsotovalero/cesarsotovalero.github.io#30] [Announcements]
# blog/use-custom-latex-macros-to-boost-your-writing-productivity

LaTeX macros allow increasing productivity by avoiding typing repetitive LaTeX commands. In this tutorial, I show you how to use them.

https://www.cesarsotovalero.net/blog/use-custom-latex-macros-to-boost-your-writing-productivity.html

**Comment by martisak** [comment]
That tip on Executing Python Scripts blew my mind! For scripts like the example, that execute really fast, this is really useful! Overleaf seems to add `--shell-escape` automatically, while this will be needed if you compile locally `latexmk -pdf --shell-escape test.tex `. 

For longer scripts, that actually take a while to run, I prefer to use Make or similar to run the script and output to a file that I then include.
